docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -890,6 +890,22 @@ Since the Key Manager offers **relay execution** via signed message, it's import
 
 - Signed messages with sequential nonces should be **executed in order**, meaning a signed message with nonce 4 can't be executed before the signed message with nonce 3. This is a critical problem which can limit the usage of relay execution.
 
+:::warning
+
+When execution 3 sequential relay calls with sequential nonces in a single channel, you need to keep in mind the fact that if the first transaction does revert, the next 2 will revert as well. That happens because a nonce is assigned to each relay call.
```

Comment:
Let's precise **relay call nonces** and Key Manager. Otherwise, newcomers could get confused with generic blockchain nonces.

```suggestion
When executing 3 sequential relay calls with sequential nonces in a single channel (= nonces from the KeyManager retrieved via [`getNonce`](./link-to-abi-function), keep in mind that **if the first transaction does revert, the next 2 will revert in turns**. That happens because a Key Manager nonce is assigned to each relay call.
```